descriptors/ModuleDescriptor-template.json

```diff
@@ -348,28 +384,30 @@
       }
     },
     "env": [
-      {
-        "name": "JAVA_OPTIONS",
-        "value": "-XX:MaxRAMPercentage=66.0"
-      },
+      { "name": "JAVA_OPTIONS", "value": "-XX:MaxRAMPercentage=66.0", "description": "Java options" },
       { "name": "CAMUNDA_BPM_ADMINUSER_EMAIL", "value": "admin@localhost", "description": "The e-mail address of the Camunda administration user." },
       { "name": "CAMUNDA_BPM_ADMINUSER_ID", "value": "admin", "description": "The account name of the Camunda administration user." },
       { "name": "CAMUNDA_BPM_ADMINUSER_PASSWORD", "value": "admin", "description": "The password of the Camunda administration user." },
       { "name": "CAMUNDA_BPM_DATABASE_SCHEMAUPDATE", "value": "true", "description": "If Camunda should auto-update the BPM database schema." },
       { "name": "CAMUNDA_BPM_METRICS", "value": "false", "description": "Enable or disable Camunda metrics by default." },
-      { "name": "DB_HOST", "value": "postgres" },
-      { "name": "DB_PORT", "value": "5432" },
-      { "name": "DB_USERNAME", "value": "folio_admin" },
-      { "name": "DB_PASSWORD", "value": "folio_admin" },
-      { "name": "DB_DATABASE", "value": "okapi_modules" },
-      { "name": "DB_QUERYTIMEOUT", "value": "60000" },
-      { "name": "DB_CHARSET", "value": "UTF-8" },
-      { "name": "DB_MAXPOOLSIZE", "value": "16" },
-      { "name": "EVENT_QUEUE_NAME", "value": "event.queue", "description": "The ActiveMQ event queue name." },
+      { "name": "DB_HOST", "value": "postgres", "description": "Postgres hostname" },
+      { "name": "DB_PORT", "value": "5432", "description": "Postgres port" },
+      { "name": "DB_USERNAME", "value": "folio_admin", "description": "Postgres username" },
+      { "name": "DB_PASSWORD", "value": "folio_admin", "description": "Postgres username password" },
+      { "name": "DB_DATABASE", "value": "okapi_modules", "description": "Postgres database name" },
+      { "name": "DB_QUERYTIMEOUT", "value": "60000", "description": "Database query timeout." },
+      { "name": "DB_CHARSET", "value": "UTF-8", "description": "Database charset." },
+      { "name": "DB_MAXPOOLSIZE", "value": "16", "description": "Database max pool size." },
+      { "name": "KAFKA_HOST", "value": "kafka", "description": "Kafka broker hostname" },
```

Comment:
This should described the `ENV` environment variable.

Reply:
What should its value be? Why not use the application default?